ipmiMgrApp/Db/cooling_unit.substitutions

```diff
@@ -23,19 +23,19 @@
 #  14   Compact  0xf0    0x1e  0x62  0x10              HS 041 CU2
 # ------------------------------------------------------------------
 
-file "fru_cu_common.db"
+file "fru_common_alias.db"
 {
     pattern { id    , fruid    }
             { $(id) , $(fruid) }
 }
 
-file "fru_common.db"
+file "fru_cu_common_alias.db"
```

Comment:
This diff is confusing because it's changing cu_common to common and vice-versa. Please reorder this block.